platform/src/test/resources/test-anchor-config.yaml

```diff
@@ -0,0 +1,98 @@
+stellar:
+  anchor:
+    # Configure the application from this file after it is loaded in memory
+    # If `config` is in-memory, the yaml file contains all settings for the server.
+    config: in-memory
+    app-config:
+      type: config-spring-property                # Activate [config-spring-property] module
+      settings: app-config                        # The location of the configuration data
+    data-access:
+      type: data-spring-jdbc                      # Activate [config-spring-jdbc] module.
+      settings: data-spring-jdbc-settings         # The location of the configuration data in this file.
+    logging:
+      type: logging-logback
+      settings: logging-logback-settings
+
+#
+# Application settings
+#
+app-config:
+  # general
+  app:
+    stellarNetworkPassphrase: Test SDF Network ; September 2015
+    hostUrl: http://localhost:8080
+    languages: en
+    horizonUrl: https://horizon-testnet.stellar.org
+    assets: assets-test.json
+    jwtSecretKey: secret
+
+  # sep-1
+  sep1:
+    enabled: true
+    stellarFile: sep1/stellar-wks.toml
+
+  # sep-10
+  sep10:
+    enabled: true
+    homeDomain: localhost:8080
+    clientAttributionRequired: false
+    clientAttributionAllowList: lobstr.co,preview.lobstr.co # use this if we want to white list
+    # clientAttributionDenyList: # use this if we want to black list.
+    authTimeout: 900
+    jwtTimeout: 86400
+    signingSeed: SAX3AH622R2XT6DXWWSRIDCMMUCCMATBZ5U6XKJWDO7M2EJUBFC3AW5X
+
+  # sep-24
+  sep24:
+    interactiveJwtExpiration: 300
+    interactiveUrl: http://localhost:8080/ref/sep24/interactive
+
+  payment-gateway:
+    #
+    # Payment Circle configurations
+    #
+    circle:
+      name: "circle"
+      enabled: true
+      circleUrl: https://api-sandbox.circle.com
+      secretKey: secret
+      horizonUrl: https://horizon-testnet.stellar.org
+      stellarNetwork: TESTNET
+
+    #
+    # Payment Stellar configurations
+    #
+    stellar:
+      enabled: false
+      name: "stellar"
+      horizonUrl: https://horizon-testnet.stellar.org
+      secretKey: secret
```

Comment:
@lijamie98, about your comment:
https://github.com/stellar/java-stellar-anchor-sdk/pull/63#issuecomment-1043344235

I think this is fine for now. Let's not overthink this and move forward... making the stellar property accept an array is something we can do later, when we really need it.